[ERROR] There is a conflict between multiple exceptions for assigning the same variable.
[ERROR] 
[ERROR] This justification is true:
[ERROR]   --> test_scope/bad/scope.catala_en
[ERROR]    | 
[ERROR] 14 |   definition b under condition not c consequence equals 0
[ERROR]    |                                ^^^^^
[ERROR]    + Article
[ERROR] 
[ERROR] This justification is true:
[ERROR]   --> test_scope/bad/scope.catala_en
[ERROR]    | 
[ERROR] 13 |   definition b under condition not c consequence equals 1337
[ERROR]    |                                ^^^^^
[ERROR]    + Article
